### DISCUSSION TITLE
AdmissionWebhooks together with SignedPolicy

### DISCUSSION THREAD

**Discussion by keeyor** [OvenMediaLabs/OvenMediaEngine#1218] [Help]
Is it possible to use AdminissionWebhooks response to genarate and return a SignedPolicy url in new_url field? I tried this already, but it seems that SignedPolicy comes in first and prevents AdminissionWebhooks to send a POST to ControlCenter. Will this scheme work if I create one Virtual Host with only AdministionWebhooks enabled and redirect the stream to another Virtual Host with only SingedPolicy enabled?

**Comment by getroot** [comment]
OME processes SignedPolicy first, then AdmissionWebhooks. However, I have yet to see a case where both are used at the same time. why do you need this? What's the use case for this?
